conanfile.py

```diff
@@ -25,6 +25,10 @@ class ImguiSFML(ConanFile):
     * imconfig_install_folder: None or String
       Use 'None' if you want this build to include imconfig.h file,
       otherwise use anything else.
+
+    * imgui_revision: None or String
+      Tag or branch of ImGui repository that should be patched with
+      ImGui-SFML. 'None' will results in master branch.
```

Comment:
Let's drop this behaviour. If the user specifies 'None', they can easily specify 'master'